src/src/components/Contexts/CurrentViewContext.tsx

```diff
@@ -9,11 +9,11 @@ const supportedViews = [
   'day',
   'week',
   'month',
-  'year' /*, 'customday'*/,
+  /*'year' , 'customday'*/
```

Comment:
Is there a reason we got rid of year here?

Reply:
The Stacked Bar Chart is not yet optimized for displaying a year view (it would display 365 columns - not good)
Instead, we should make it group all data by months so that there are only 12 columns.

Also, the events endpoint has a limit of 2500 events - beyond that, there is pagination, but our events fetcher code does not handle pagination yet, thus not all events would be fetched.

And even if all events were fetched, it would probably take some time to fetch them all, thus there perhaps should be a loading indicator of some sort.

Reply:
All of these problems are solvable, but not for this sprint. Thus, it's better to disable the year view for now, although I am very excited to bring it back once it is ready as it would be very useful